pom.xml

```diff
@@ -1388,7 +1388,8 @@
 
     <jms.url>tcp://localhost:61616</jms.url>
     <activemq.version>5.6.0</activemq.version>
-    <!-- End es and camel stuffs -->
+
+    <bot.regexpFilter>.*(bot|crawler|baiduspider|80legs|ia_archiver|voyager|yahoo! slurp|mediapartners-google|Linguee Bot|SemrushBot).*</bot.regexpFilter>
```

Comment:
AllRequestsInterceptor.java also has default values defined.  I would expect this to be optional.
Maybe an empty template <bot.regexpFilter></bot.regexpFilter>

However when I tried to remove this line expecting it to use the default however it failed due to circular reference.
org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'apiMvcInterceptor' defined in ServletContext resource [/WEB-INF/spring-servlet.xml]: Circular placeholder reference 'bot.regexpFilter' in property definitions; nested exception is java.lang.IllegalArgumentException: Circular placeholder reference 'bot.regexpFilter' in property definitions

Reply:
>  it was odd to have the default settings in 2 locations

Yep.

> I suggest that we use the new the new syntax for the maven resource
bot.regexpFilter=@bot.regexpFilter@

Indeed. LGTM

> Note: I will be sending a pull request to your branch so you can see the changes.

Thanks.